docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -377,12 +377,15 @@ above, this SQL will delete all partitions for which ``country`` is ``US``::
     DELETE FROM iceberg.testdb.customer_orders
     WHERE country = 'US'
 
-Currently, the Iceberg connector only supports deletion by partition.
-This SQL below will fail because the ``WHERE`` clause selects only some of the rows
-in the partition::
+Both v1 and v2 tables will perform a partition delete if the ``WHERE`` clause
+meets these conditions.
 
-    DELETE FROM iceberg.testdb.customer_orders
-    WHERE country = 'US' AND customer = 'Freds Foods'
+Row level deletion
+^^^^^^^^^^^^^^^^^^
+
+Tables using v2 of the Iceberg specification support deletion of individual rows
+using the ``merge-on-read`` mode. In order to use this feature the ``format_version``
```

Comment:
merge-on-read in monospace looks like we support deletes only if deletion mode is merge-on-read, while we ignore it

let's not use the merge-on-read in a code block. Let's say we create deletion files.